README.md

```diff
@@ -2,69 +2,70 @@
 
 [![Build Status SVG][]][Repo on GitHub Actions]
 [![OpenSSF Scorecard SVG][]][Scorecard Results]
-[![first-timers SVG][]][first-timers]
 
-The https://dart.dev site, built with [Jekyll][] and hosted on [Firebase][].
+The documentation site for the [Dart programming language](https://dart.dev), 
+built with [Eleventy][] and hosted on [Firebase][].
 
-[We welcome contributions](CONTRIBUTING.md),
-and we're [first-timer friendly][first-timers]!
+We welcome contributions of all kinds!
+To set up the site locally, follow the
+below guidelines on [Building the site](#build-the-site).
+To learn more about contributing to this repository,
+check out the [Contributing guidelines](CONTRIBUTING.md).
 
 ## Getting started
 
 Start by looking for an [issue](https://github.com/dart-lang/site-www/issues)
 that catches your interest, or create an issue with your proposed change.
-Ask for the issue to be assigned to you.
+Consider adding a comment to let everyone know that you're working on it, and
+feel free to ask any questions you have on the same issue.
 
-To update this site, fork the repo, make your changes, and generate a pull
-request. For simple changes (such as to CSS and text), you probably don't need
-to build this site. Often you can make changes using the GitHub UI.
+To update this site, fork the repo, make your changes,
+and generate a pull request.
+For simple changes (such as style and typo fixes),
+you probably don't need to build this site.
+Often you can make changes using the GitHub UI.
+We can stage the changes automatically in your pull request.
```

Comment:
Great call out!

Reply:
It was just an adjustment of what was there, but thanks :P

- While the new setup is easier, I still want to discourage developers looking to quickly contribute.